personio-personnel-data-api-oa3.yaml

```diff
@@ -719,7 +719,7 @@ paths:
           schema:
             type: string
             format: date
-        - name: employees
+        - name: employees[]
```

Comment:
Can't this be a breaking change for some integrations and consumers?

Reply:
It won’t break any integrations as this was a flaw in the API documentation, and the actual query param was always `employees[]`  and not `employees`.

Reply:
Previously if someone tried to use the "Try out" function in the API doc, it wouldn't have worked as expected, but now it will since the params have been fixed. This is the only impact of the change.